requester.py

```diff
@@ -0,0 +1,99 @@
+import requests
+
+# For now request works only for default '24h' forecast period
+
+# weatherapi.com API
+
+
+def request_weatherapi(keys_dict, args):
+    url_wheatherapi = 'https://api.weatherapi.com/v1/forecast.json'
+    API_KEY_wheatherapi = keys_dict['wheatherapi.com']
+
+    params_wheatherapi = dict(key=API_KEY_wheatherapi, q=args.location, days=1)
+    resonse_wheatherapi = requests.get(url_wheatherapi, params=params_wheatherapi)
+
+    # check if lack of proper request response is due to 403 error
+    if resonse_wheatherapi.status_code == 403:
+        print("Supplied API key for wheatherapi.com is wrong.")
+        exit()
+
+    resonse_wheatherapi_dict = resonse_wheatherapi.json()
+    resonse_wheatherapi_hours = resonse_wheatherapi_dict['forecast']['forecastday'][0]['hour']
+
+    # extracting needed information from request response
+    hours_wheatherapi = []
+    temp_c_wheatherapi = []
+    rain_wheatherapi = []
+    for item in resonse_wheatherapi_hours:
+        hours_wheatherapi.append(item["time"])
+        temp_c_wheatherapi.append(item["temp_c"])
+        rain_wheatherapi.append(item['precip_mm'])
+
+    # get latitute and longitute of the location
+    latitude = resonse_wheatherapi_dict['location']['lat']
+    longitude = resonse_wheatherapi_dict['location']['lon']
+
+    weatherapi_dict = {'hours': hours_wheatherapi, 'temp': temp_c_wheatherapi,
+                       'rain': rain_wheatherapi, 'latitude': latitude, 'longitude': longitude}
+
+    return weatherapi_dict
+
+# open-meteo.com API
+
+
+def request_openmeteo(latitude, longitude):
```

Comment:
Why in this and the next function there's no error response handling?

Reply:
Yes, it is true. Honestly, I thought I could polish the code when the whole program worked as it should for basic parameters. I will get back to it in the future.

Reply:
I get your point but good practice would be to do it upfront, all this kind of polishing should be done at the implementation phase, especially since you already done it for one method. Let's discuss it at the upcoming meeting.

Reply:
@KrzysiekJodlowski I added error 403 handling to the request_meteostat() function. Request_openmeteo() function does not require an API key, so do not need error 403 handling.

Reply:
Ok, cool, as for the `request_openmeteo()` let it be if it works as you say. Still, I see the logic for handling the `403` is pretty much the same, can you please extract it to some common method? Also, it would be good to handle at least [the `503` error also](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503), so when the server is down. This can happen always e.g. because of some maintenance time. With having one method for error handling this should be a matter of adding just 3 other lines of code.

Reply:
@KrzysiekJodlowski Done. I created function **response_errors(**api_response, name**)** that checks 403 and 503 errors. I used it in all three API request functions.

Reply:
@deerhunter4 sounds good but I cannot see any code changes?

Reply:
@KrzysiekJodlowski Now you should see